flask_resty/sorting.py

```diff
@@ -15,14 +15,14 @@ def sort_query(self, query, view):
 
 class FieldSortingBase(SortingBase):
     def sort_query(self, query, view):
-        field_orderings = self.get_request_field_orderings()
+        field_orderings = self.get_request_field_orderings(view)
         return self.sort_query_by_fields(query, view, field_orderings)
 
     def sort_query_by_fields(self, query, view, field_orderings):
         criteria = self.get_criteria(view, field_orderings)
         return query.order_by(*criteria)
 
-    def get_request_field_orderings(self):
+    def get_request_field_orderings(self, view):
```

Comment:
This change is unnecessary, but I think it's probably better to have this anyway.

Reply:
@itajaja Thoughts on whether this change makes sense?

Reply:
I guess it's ok, also ok to add it later when needed. up to you